DigitalLearningSolutions.Data.Tests/Services/SectionContentServiceTests.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace DigitalLearningSolutions.Data.Tests.Services
+{
+    using DigitalLearningSolutions.Data.Models.SectionContent;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Data.Tests.Helpers;
+    using FakeItEasy;
+    using FluentAssertions;
+    using Microsoft.Extensions.Logging;
+    using NUnit.Framework;
+
+    internal class SectionContentServiceTests
+    {
+        private SectionContentService sectionContentService;
+
+        [SetUp]
+        public void Setup()
+        {
+            var connection = ServiceTestHelper.GetDatabaseConnection();
+            var logger = A.Fake<ILogger<SectionContentService>>();
+            sectionContentService = new SectionContentService(connection, logger);
+        }
+
+        [Test]
+        public void Get_section_content_should_return_section_content()
```

Comment:
I think it would be useful to test a few more cases. For example testing that a section which has percentage complete is fetched correctly 🙂